md/applicationCreation.md

```diff
@@ -0,0 +1,70 @@
+# Application Descriptor
+
+
+
+## Overview
+
+In Bonita BPM 7.5, we introduced the possibility to manage application descriptors from the Studio.  
+It accounts for an additional step to ease the management of [Living Applications](applications.md) in Bonita BPM, since our objective is to offer a unique application deployment artifact, managed from the Studio.  
+Therefore, all resources used by applications will be centralized and shared by the Bonita BPM development suite: Bonita BPM Studio and the UI Designer.  
+Those resources are used to create Application Descriptors from the Studio. Application Descriptors can be deployed in the Portal to create Applications.  
+
+## Definition 
+
+Basically, an **application descriptor** is an XML file. It represents the skeleton of an Application, as it contains references to resources installed in Bonita BPM Portal and used by a given application.  
+An application descriptor must be imported onto the portal (Administrator profile), in the Applications tab.  
+Before Bonita BPM 7.5, this portal tab was the only way to create and manage applications.  
+For development purposes, the studio can deploy applications descriptors onto the portal for you (_only for Performance and Efficiency editions_).
+
+Application descriptors are packaged in **application files**.  
+An **application file** is a XML file containing one or more application descriptors.  
+
+	 <!--  Application File -->
+    <applications>
+		<application>
+			<!--  First Application Descriptor -->
+		</application>
+		<application>
+			<!--  Second Application Descriptor -->
+		</application>
+	</applications>
+	
+Application Descriptors in a given application file are totally **independent**.  
+The purpose of grouping application descriptors in one file is to manage together applications with the same company purpose, each of them dedicated to a different user profile.  
+For example, four leave management applications for employees, managers, Human Resources officers, and Administrators should be managed together, therefore grouped in the same application file.  
+
+## How to create and build an Application Descriptor
```

Comment:
First you create it, then you build it by adding menu pages...
No need to change this word in my view.